modules/ducktests/tests/ignitetest/services/utils/ignite_aware_app.py

```diff
@@ -106,7 +108,23 @@ def env(self):
         return "export MAIN_CLASS={main_class}; ".format(main_class=self.servicejava_class_name) + \
                "export EXCLUDE_TEST_CLASSES=true; " + \
                "export IGNITE_LOG_DIR={log_dir}; ".format(log_dir=self.PERSISTENT_ROOT) + \
-               "export USER_LIBS=%s:/opt/ignite-dev/modules/ducktests/target/*; " % self.user_libs
+               "export USER_LIBS=%s:/opt/ignite-dev/modules/ducktests/target/*%s; " % \
+               (self.user_libs, self.app_service_libs())
+
+    def app_service_libs(self):
+        """
+        :return: Libs required to start IgniteAwareApplication java implementation.
+        """
+        if self.version.is_dev:
+            return ""
+
+        libs = ""
+
+        for line in self.nodes[0].account.ssh_capture(
+                "ls -d %s/libs/optional/ignite-aws/* | grep jackson | tr '\n' ':' | sed 's/.$//'" % self.path.home()):
```

Comment:
Why do you extract jackson jars from the AWS library? As I can see AWS isn't used in the application

Reply:
Because we should gain it somewhere, but it always presented as a dependency and always have the same version for all cases (set inside parent.pom)

Reply:
Why do not keep them in pom.xml of ducktests module?

Reply:
For every old version?

Reply:
Oh, now I understand. But do we need use strict link to AWS project, as we can't guarantee that it wouldn't be changed in future? Could we just use the `find` util to locate jackson jars anywhere in optional libs?

Reply:
Also there must be a comment with description of solution

Reply:
Comment added. Can we postpone ideal resolving (locate jackson jars anywhere in optional libs) and just use AWS as source for now?

Reply:
Ok, I'd think it should be part of infrastructure spec.